java/README.md

```diff
@@ -6,6 +6,7 @@
 Please refer to the [Dataproc Templates (Java - Spark) README](java/README.md) for more information.
 
 * [BigQueryToGCS](src/main/java/com/google/cloud/dataproc/templates/bigquery#BigQuery-To-GCS)
```

Comment:
We also need to add this at main home page [READEME.md](https://github.com/GoogleCloudPlatform/dataproc-templates?tab=readme-ov-file)